elastic/tests/compose/docker-compose.yaml

```diff
@@ -4,12 +4,15 @@ services:
   elastic:
     image: "${ELASTIC_REGISTRY}:${ELASTIC_IMAGE}"
     environment:
-      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - "ES_JAVA_OPTS=-Xms1g -Xmx1g"
       - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m"
       - "cluster.name=test-cluster"
       - "node.name=test-node"
       - "discovery.type=single-node"
-      - "path.repo=[\"/backup/my_backup_location\"]"
+      - "path.repo=[\"/usr/share/elasticsearch/data\"]"
+      - "xpack.security.enabled=false"
```

Comment:
These are to disable security, as ES8 doesn't use opendistro anymore.

Reply:
Could you add the reason as an inline comment?